spec/config/initializers/rack_attack_spec.rb

```diff
@@ -0,0 +1,122 @@
+# require 'rails_helper'
```

Comment:
This specs are examples. They have very specific configurations and they should be configured for each project individually. That why they are commented and shown as examples with the explanation of how to configured them.